styleguide/derek/templates/content-driven/certification.ejs

```diff
@@ -0,0 +1,22 @@
+<div class="container standard-padding-full">
+  <div class="row">
+    <div class="col-sm-9">
+      <div class="subpanel">
+        <div class="subpanel-inner">
+          <%- partial('../../layouts/leadFullWidth.ejs') %>
```

Comment:
This is already inside a container. Use the partial instead of the layout here instead.